kubert/src/runtime.rs

```diff
@@ -553,3 +589,14 @@ impl LogSettings {
         self.format.try_init(self.filter)
     }
 }
+
+// === impl RuntimeMetrics ===
+
+impl RuntimeMetrics {
+    /// Creates a new set of metrics and registers them.
+    #[cfg(feature = "prometheus-client")]
+    pub fn new(registry: &mut Registry) -> Self {
+        let watch = ResourceWatchMetrics::new(registry);
```

Comment:
```suggestion
        let watch = ResourceWatchMetrics::new(registry.sub_registry_with_prefix("watch"));
```

Because the user only provides a high-level prefix, we want to scope our metrics internally so that we can add other scopes later. I think 'watch' is appropriate because it matches the Runtime::watch API.